src/api.rs

```diff
@@ -108,6 +107,41 @@ pub fn feed_player(
     Ok(JSON(FeedMeResponse { score }))
 }
 
+#[derive(Debug, Serialize)]
+pub enum NoseGoesResponse {
+    Survived,
+    Died,
+}
+
+#[post("/nose-goes/<id>")]
+pub fn nose_goes(
+    id: PlayerId,
+    nose_goes: State<NoseGoesState>,
+) -> Result<JSON<NoseGoesResponse>> {
+    let mut nose_goes = nose_goes.lock().expect("Nose-goes state was poisoned!");
+    match *nose_goes {
+        NoseGoes::Inactive { .. } => {
```

Comment:
It will trigger if the client is too slow, but it won't corrupt any game state. It'll return an error to the client, but the client will pretty much ignore it.